### PR TITLE
Add a wheel client api

### PR DESCRIPTION
**Pull request by goodboy**
A handy `wheel` client like already exists for `runner` and `local` :)

I did a couple cleanups with the import list and some white space.
Hope that's ok ;)

### CODE REVIEW

File: pepper/libpepper.py [hunk 1]
```diff
@@ -361,6 +358,27 @@ def runner(self, fun, **kwargs):
...
+          wheel('key.accept', match='myminion')
+        '''
+        low = {
+            'client': 'wheel',
+            'fun': fun,
+        }
+
+        if arg:
+            low['arg'] = arg
+        # if kwarg:
```
Comment (goodboy) [inline pepper/libpepper.py]: The [`wheel` module(s)](https://docs.saltstack.com/en/latest/ref/wheel/all/salt.wheel.key.html#module-salt.wheel.key) do take `kwargs` so I should include this right?

**Comment by goodboy** [comment]
@whiteinge any opinion on this? 
If it's not something you guys want that's all cool :)
Just wondering if I should keep it on my backlog.

File: pepper/libpepper.py (hunk 1 above)
Comment (whiteinge) [reply pepper/libpepper.py]: WheelClient's cmd_sync/cmd_async methods definitely accept kwargs. I kind of remember they do **not** accept positional args though...

**Comment by whiteinge** [comment]
Slow reply! Sorry.

Yeah, great addition. Thanks for the cleanups too. Uncomment the kwargs and this looks good to merge.

I don't think support for positional args was ever wired up in WheelClient. But including this won't hurt anything and should Just Work (TM) when that is added to WheelClient.

**Comment by goodboy** [comment]
@whiteinge done!

Yeah I haven't tested the the `kwargs` unfortunately (do you want me to?) but I wouldn't be surprised if you're right because I definitely can't get it to work for the `runner` client :(

Anyway glad to help and thanks for accepting :)

**Review by whiteinge — APPROVED** [review]
Thanks!

**Pull request merged**